app/src/main/java/com/example/campuscollab/view/CreateProjectFragment.java

```diff
@@ -39,20 +51,23 @@ public View onCreateView(
         projectNameInput = binding.projectName;
         projectDescriptionInput = binding.projectDescription;
         groupMemberNumber = binding.groupSizeInput;
+        groupMemberNumber.setOnItemSelectedListener(this);
+
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this.getContext(), R.array.group_size_options, R.layout.custom_spinner_element);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+
+        groupMemberNumber.setAdapter(adapter);
 
         createProjectButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 if (isValidProject()) {
-                    //TODO add access of actual current user email through one of the services
-                    //TODO update declaration to include number of people added
-
-                    Integer numMembers = Integer.parseInt(groupMemberNumber.getText().toString());
-
-                    Project newProject = new Project(projectNameInput.getText().toString(), "currentuseremail",
-                                                     projectDescriptionInput.getText().toString(), new String[] {});
+                    Project newProject = new Project(projectNameInput.getText().toString(), authService.getCurrentUser().getUid(),
+                                                     projectDescriptionInput.getText().toString(), new ArrayList<>(),
+                                                     new Timestamp(Timestamp.now().toDate()), new Timestamp(Timestamp.now().toDate()), maxGroupSize);
```

Comment:
Could we simplify this by just using { ...., Timestamp.now(), Timestamp.now(), ...}
Timestamp.now() returns a new timestamp of the current system time, so we shouldn't need to convert it to a date and then back to a timestamp again

Reply:
Yeah, good catch.  I fixed it.

Reply:
Thanks!